tests/golden/check-local-refs/expected2.gold

```diff
@@ -1,145 +1,127 @@
 === Invalid references found ===
 
-  ➥  In file dir1/dir2/d2f1.md
+  ➥  In file dir2/d2f1.md
      bad reference (file-local) at src:9:1-18:
        - text: "bad-cf-ref"
        - link: -
        - anchor: bad
 
      Anchor 'bad' is not present
 
-  ➥  In file dir1/dir2/d2f1.md
+  ➥  In file dir2/d2f1.md
      bad reference (relative) at src:27:1-43:
        - text: "bad-file-rel"
        - link: ../a/b/c/unexisting-file.md
        - anchor: -
 
      File does not exist:
-       dir1/dir2/../a/b/c/unexisting-file.md
+       a/b/c/unexisting-file.md
 
-  ➥  In file dir1/dir2/d2f1.md
-     bad reference (relative) at src:28:1-31:
-       - text: "bad-casing-file-rel"
-       - link: D2F2.md/
-       - anchor: -
-
-     File does not exist:
-       dir1/dir2/D2F2.md/
-
```

Comment:
This kind of output differences appear because the GitHub flavour is now case-insensitive.